code/ship/ship.cpp

```diff
@@ -10334,6 +10334,8 @@ int ship_fire_primary(object * obj, int stream_weapons, int force)
 				if (winfo_p->wi_flags3 & WIF3_APPLY_RECOIL){
 					firepoint_list = new vec3d[numtimes * points];
 					vm_vec_zero(&total_impulse);
+				} else {
+					firepoint_list = nullptr;
```

Comment:
What version of clang is Xcode using? nullptr is supposed to be supported from 3.0 according to [this page](http://clang.llvm.org/cxx_status.html)